tools/cordova/builder.js

```diff
@@ -392,23 +392,23 @@ export class CordovaBuilder {
 
   configureAndCopyResourceFiles(resourceFiles, iosElement, androidElement) {
     _.each(resourceFiles, resourceFile => {
-      // Copy file in cordova project root directory
-      var filename = path.parse(resourceFile.src).base;
+      // Copy resource files in cordova project root ./resource-files directory keeping original absolute path
+      var filepath = files.pathResolve(this.projectContext.projectDir, resourceFile.src);
       files.copyFile(
-        files.pathResolve(this.projectContext.projectDir, resourceFile.src),
-        files.pathJoin(this.projectRoot, filename));
+        filepath,
+        files.pathJoin(this.projectRoot, "/resource-files/", filepath));
       // And entry in config.xml
       if (!resourceFile.platform ||
           (resourceFile.platform && resourceFile.platform === "android")) {
         androidElement.element('resource-file', {
-          src: resourceFile.src,
+          src: files.pathJoin("./resource-files/", filepath),
```

Comment:
Will it work without the "./" here and on line 411? This would make the chances higher of it working on windows... I can't test it on windows, but AFAIK the meteor fs code doesn't seem to be dealing with this POSIX convention. It seems to be only dealing with slashes. Nice work btw 👍 thanks!

```suggestion
          src: files.pathJoin("resource-files/", filepath),
```